ebos/eclgenericoutputblackoilmodule.cc

```diff
@@ -762,6 +795,47 @@ assignToSolution(data::Solution& sol)
         // reallocated)
         tracerConcentrations_.resize(0);
     }
+
+    // Flows
+    if (!flowsi_[gasCompIdx].empty())
+        sol.insert("FLOGASI+", UnitSystem::measure::gas_surface_rate, flowsi_[gasCompIdx], data::TargetType::RESTART_SOLUTION); 
```

Comment:
I think @atgeirr mentioned this in the earlier review, but I didn't see a response.  Is there a reason you chose to use separate logic for the flow arrays here instead of integrating these into the output system using the [existing](https://github.com/OPM/opm-simulators/blob/38282e30f7d3fa8a97cce3f8bc5efe07bad5dbb9/ebos/eclgenericoutputblackoilmodule.cc#L666) table of arrays?